crates/noirc_evaluator/src/ssa.rs

```diff
@@ -62,7 +62,8 @@ pub(crate) fn optimize_into_acir(
             .dead_instruction_elimination()
             .print(print_ssa_passes, "After Dead Instruction Elimination:");
     }
-    ssa.into_acir(brillig, abi_distinctness)
+    let array_use = ssa.array_use();
```

Comment:
Can we rename this to something more descriptive like `find_last_array_uses`?

Reply:
Done